src/test/java/edu/kis/powp/jobs2d/TestJobs2dPatterns.java

```diff
@@ -25,8 +28,8 @@ public class TestJobs2dPatterns {
 	private static void setupPresetTests(Application application) {
 		SelectTestFigureOptionListener selectTestFigureOptionListener = new SelectTestFigureOptionListener(
 				DriverFeature.getDriverManager());
-
 		application.addTest("Figure Joe 1", selectTestFigureOptionListener);
+		application.addTest("Figure Joe 2", e -> FiguresJoe.figureScript2(DriverFeature.getDriverManager().getCurrentDriver()));
```

Comment:
It is good to keep a convention used in the original code (class instance instead of lambda)